tests/conftest.py

```diff
@@ -0,0 +1,48 @@
+import socket
+
+import pytest
+from airflow.utils import timezone
+from airflow.utils.db import create_session
+from airflow.utils.state import State
+from pytest_socket import disable_socket, SocketBlockedError
+
+from tests.factories import DAGFactory, PythonOperatorFactory
+
+
+def pytest_runtest_setup():
+
+    def block_lookup(*args, **kwargs):
+        raise SocketBlockedError
+
+    disable_socket()
+    socket.getaddrinfo = block_lookup
```

Comment:
if this comes from Boto idiosyncrasies, state it in the docstring of this function

Reply:
description added.